File: serv/src/main/java/task_manager/server/models/Task.java
package task_manager.server.models;

/**
 * Created by Alex on 19.02.2018.
 */
import com.fasterxml.jackson.annotation.JsonProperty;


public class Task {
    @JsonProperty("id")
    private int id;
    @JsonProperty("author_id")
    private int author_id;
    @JsonProperty("caption")
    private String caption;
    @JsonProperty("about")
    private String about;

    public Task(int id, int author_id, String caption, String about) {
        this.id = id;
        this.author_id = author_id;
        this.caption = caption;
        this.about = about;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getAuthorId() {
        return author_id;
    }

    public void setAuthorId(int id) {
        this.author_id = id;
    }

    public String getCaption() {
        return caption;
    }

    public void setCaption(String caption) {
        this.caption = caption;
    }

    public String getAbout() {
        return about;
    }

    public void setAbout(String about) {
        this.about = about;
    }
}

File: serv/src/main/kotlin/task_manager/server/models/Task.kt
package task_manager.server.models;

/**
 * Created by Alex on 19.02.2018.
 */
import com.fasterxml.jackson.annotation.JsonProperty;
//import com.sun.org.apache.xpath.internal.operations.Bool;

data class Task(@JsonProperty("id")
                var id: Int?,
                @JsonProperty("author_id")
                var author_id: Int?,
                @JsonProperty("caption")
                var caption: String?,
                @JsonProperty("about")
                var about: String?,
                @JsonProperty("checked")
                var checked: Boolean?) {
    constructor(author_id: Int?, caption: String?, about: String?, checked: Boolean?)
            : this(0, author_id, caption, about, checked)
}
